test/runtests.jl

```diff
@@ -194,6 +194,41 @@ facts("FactCheck assertion helper functions") do
     end
 end
 
+function lineFromOutput(output)
+    answer =  match(r"line:(\d+)", output)
+    answer == nothing? output: parse(Int, answer[1])
+end
+
+if VERSION >= @v_str("0.5-")
```

Comment:
@yuyichao There is no reason to not use version literal. I have just replaced it. Should I squash the commits?